### DISCUSSION TITLE
Moving official season 2

### DISCUSSION THREAD

**Discussion by Dynnammo** [mawoka-myblock/ClassQuiz#46] [General]
I may not be as informed as I should, but here is some facts I would like to discuss : ClassQuiz seems right now, even if it's not an old project, the only free alternative to Kahoot! and similar engines. A quick search on any search engines returns a lot of other proprietary softwares, but not a single one as open as ClassQuiz.

Other point : right now, the project is supported by it's founder, alias @mawoka-myblock. It would be shame we stay in this situation, as a project that depends on a few has a high risk of discontinuation.  Therefore, building a community around it is a short/middle-term task we could handle.

For all those reasons, I would consider the following : 
1. **Building a community** : with discussion tools like Discord or  Element, in order to provide a space for discussion between developers but also people wanting to contribute or discuss their problems with the application, since it's still in early stage. Some examples : https://github.com/nocodb/nocodb (has a Discourse + Discord), Airbyte (Slack + Discourse)
2. **Moving official on Github** : using the same examples, moving the repository to an organization user, so having `classquiz/ClassQuiz` instead of `mawoka-myblock/ClassQuiz` to centralize all repos about the project (main site, futur blog if it exists, side projects, add-ons/community plugins etc. on `classquiz/blog`, `classquiz/darktheme`, `classquiz/docs`, ...)

Thoughts ?

**Comment by mawoka-myblock** [comment]
That would be awesome, but the userbase is way too small for an active community. I would prefer matrix/element (space/s) over Discord. Discourse is possible but also not necessary since the community is too small. To the own github-organisation: I don't think it's needed. ClassQuiz shouldn't **need** to be splitted into different repos and moving... Is it even necessary? I know, better now than later, but still.

**Reply by mawoka-myblock** [reply]
I also don't have the resources to run an active discourse. Everything runs on a single server without backups. I just don't have the resources for that. (Moneywise)
